org-formation/organization.yaml

```diff
@@ -27,6 +27,7 @@ Organization:
         - !Ref ScipoolProdAccount
         - !Ref ScicompAccount
         - !Ref SandboxAccount
+        - !Ref NlpSandboxAccount
```

Comment:
This would place the account in the `science` OU, which might not exist depending on the state of Khai's reorg: https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/123

This should probably be in the `ScienceDevOU`?

Reply:
PR #123 has been merged. please update PR and do as @ahayden suggested